locale/Makefile

```diff
@@ -50,14 +50,14 @@ tx-pull: $(EDITFILES)
 
 tx-update: tx-pull
 	@echo
-	@echo Run rake plugin:gettext[$(DOMAIN)] from the Foreman installation, then make -C locale mo-files to finish
+	@echo Run rake plugin:gettext[$(DOMAIN)] from the Foreman installation. \
+		If this is for a release, run make -C locale mo-files from the Katello directory to finish. \
+		.mo files should not be uploaded to GitHub, they should only be packed into the gem.
 	@echo
 
 mo-files: $(MOFILES)
-	git add $(POFILES) $(POTFILE) ../locale/*/LC_MESSAGES
```

Comment:
I think the PO files should still be committed? Otherwise people without access to TX can't recreate the gem (as git now would contain some old PO with old translations)?

Reply:
An alternative approach I've considered is a separate `build-mo-files` task, but failed to submit this as a PR earlier. https://github.com/theforeman/foreman_plugin_template/pull/44 has lived as local change on my disk for too long.

Then in packaging we would call `make -C locale build-mo-files` (and resolve https://projects.theforeman.org/issues/35789). I just haven't figured out how to do that exactly. We probably need to pull in some additional build time dependencies (like `gettext`).

It was my intention to play with this, but didn't find the time.

Reply:
I always `git add` the PO files myself, I'll double check that there's something about this in tool belt. We haven't used the mo-files command for ages because git would complain about the gitignore.

Reply:
I'm going to split out two commands, build-mo-files and then po-files

Reply:
https://github.com/Katello/katello/pull/10574

https://github.com/theforeman/tool_belt/pull/542